src/symbol.js

```diff
@@ -72,36 +72,39 @@ export const getSymbolId = (masterName: string): string => {
   return symbolId;
 };
 
-const injectSymbols = () => {
+export const injectSymbols = () => {
   const globalContext = context; // eslint-disable-line
   const pages = globalContext.document.pages();
   const array = msListToArray(pages);
 
-  const symbolsPage = globalContext.document
-    .documentData()
-    .symbolsPageOrCreateIfNecessary();
-
-  let left = 0;
-  Object.keys(mastersNameRegistry).forEach((key) => {
-    const symbolMaster = mastersNameRegistry[key];
-    symbolMaster.frame.y = 0;
-    symbolMaster.frame.x = left;
-    left += symbolMaster.frame.width + 20;
-
-    const newLayer = fromSJSONDictionary(symbolMaster);
-    layers[symbolMaster.symbolID] = newLayer;
-  });
+  if (mastersNameRegistry !== null) {
+    // if mastersNameRegistry is not an object then makeSymbol was not called
+    const symbolsPage = globalContext.document
+      .documentData()
+      .symbolsPageOrCreateIfNecessary();
+
+    let left = 0;
+    Object.keys(mastersNameRegistry).forEach((key) => {
+      const symbolMaster = mastersNameRegistry[key];
+      symbolMaster.frame.y = 0;
+      symbolMaster.frame.x = left;
+      left += symbolMaster.frame.width + 20;
+
+      const newLayer = fromSJSONDictionary(symbolMaster);
+      layers[symbolMaster.symbolID] = newLayer;
+    });
 
-  // Clear out page layers to prepare for re-render
-  resetLayer(symbolsPage);
+    // Clear out page layers to prepare for re-render
+    resetLayer(symbolsPage);
 
-  renderLayers(Object.keys(layers).map(k => layers[k]), symbolsPage);
+    renderLayers(Object.keys(layers).map(k => layers[k]), symbolsPage);
 
-  let notSymbolsPage = array.find(p => String(p.name()) !== 'Symbols');
-  if (!notSymbolsPage) {
-    notSymbolsPage = globalContext.document.addBlankPage();
+    let notSymbolsPage = array.find(p => String(p.name()) !== 'Symbols');
+    if (!notSymbolsPage) {
+      notSymbolsPage = globalContext.document.addBlankPage();
+    }
+    globalContext.document.setCurrentPage(notSymbolsPage);
```

Comment:
shouldn't we store the current page at the beginning of the `injectSymbols` method and go back to it instead getting the first one here?